scratch/copy-mk.py

```diff
@@ -56,6 +56,7 @@ def parse_args(argv: Sequence[str] | None = None) -> argparse.Namespace:
         help="Pass development options in the config. Use comma separation, or omit the arg to enable all.",
     )
     parser.add_argument("--image-override", action="append", metavar="NAME:IMAGE:TAG", help="Override a single image")
+    parser.add_argument("--controller", help="Hostname of the SDP master controller")
```

Comment:
```suggestion
    parser.add_argument("--controller", help="Hostname of the CBF master controller")
```

Reply:
Either give it a default, make it a positional argument, or make it a required argument. Currently if the user doesn't pass it, you'll be trying to pass None to os.execv.

Reply:
Good point. I didn't think of making it required.

I find more than one positional argument to be confusing, so I gave this one the `--` but it's a good point about the user not putting it in. I did encounter that NoneType issue but I realised what I'd done and just re-ran the script.